src/main/java/com/michalsydoryk/app/ui/ConsoleUI.java

```diff
@@ -0,0 +1,85 @@
+package com.michalsydoryk.app.ui;
+
+import com.michalsydoryk.app.board.Board;
+import com.michalsydoryk.app.board.Board2D;
+import com.michalsydoryk.app.boarddrawer.BoardDrawer;
+import com.michalsydoryk.app.boarddrawer.ConsoleBoard2DDrawer;
+
+import java.io.PrintStream;
+import java.util.ResourceBundle;
+import java.util.Scanner;
+
+public class ConsoleUI implements UI{
+    PrintStream output;
+    Scanner input;
+    ResourceBundle resourceBundle = null;
+    BoardDrawer consoleBoard2DDrawer;
```

Comment:
Access modifiers are not necessary here?

Reply:
you right.